### DISCUSSION TITLE
Netboot hangs when interacting with the menu

### DISCUSSION THREAD

**Discussion by MasterBootRecord32** [antonym/netboot.xyz#1208] [General]
**Describe the bug**
Netboot hangs when trying to interact with the menu on old hardware (infos at the end of the report).

**To Reproduce**
1. Boot up the computer ;
2. Press F12 to access boot menu ;
3. Start Netboot from the USB stick ;
4. The system successfully configures the ethernet connection ;
5. The menu appears on screen and the countdown before booting from HDD is decreasing ;
6. Press a key on the keyboard ;
7. The system hangs and you are unable to interact with the computer anymore.  

**Expected behavior**
When pressing arrow keys, the option below should be highlighted.

**Video**
https://user-images.githubusercontent.com/117741516/220429432-e51ef17f-552a-4882-9fff-0cf24248bccf.mp4

**Additional context**
I am trying to install a new OS using Netboot on 2011 hardware : 
Toshiba Satellite L755D using AMD A6-3400M processor and Radeon HD 6520G.
Stock 2011 Legacy Phoenix bios.

**The solutions I tried**
Testing the USB key on another USB port : same problem than before.
Testing the USB key on another laptop : Netboot works perfectly fine.
Flash again the USB key using the latest official Netboot version : same problem than before.
Let the countdown reach 0 and let it boot from HDD : the system reboot straight into Netboot without trying to boot on HDD.

**Comment by antonym** [comment]
Converting to discussions as it's not really a netboot.xyz project bug. Only thing I can suggest is that it may not like the keyboard built in, you could try attaching a USB keyboard to see if that helps any or play around with the settings in the BIOS to see if there is anything that can be tuned.

I'd raise it upstream with iPXE or see if other people have has similar issues with that hardware. You might also try just using iPXE by itself to see if you get any different behavior.

**Reply by MasterBootRecord32** [reply]
Thanks for your solution.
I tried plugging an external keyboard but the issue is the same, even with playing with the settings of the fairly limited bios.
However, I tried another solution that worked for me : I burned a DVD with the Netboot official ISO and it seems to work perfectly fine. I managed to install Debian on the computer without any problem.
